### ISSUE TITLE
Paned Browser: remove «Select All» button

### ISSUE THREAD

**Issue by timpulver** [quodlibet/quodlibet#1838]
<img width="331" alt="bildschirmfoto 2016-02-20 um 20 32 06" src="https://cloud.githubusercontent.com/assets/1055819/13198732/0e26ebc8-d811-11e5-9927-c6894573c480.png">

The `Select All`-button not not necessary imo. It’s easy to just click `All` in list view or press `cmd + a`. Having the `Select All`-button just clutters the interface.

**Comment by nodiscc** [comment]
Hi, I use this button quite often - when I want to clear _all_ current filters from the left browser panes. It is much quicker than scrolling all the way top to each panel, and clicking `All` in each one. Unless I am doing something wrong, please consider keeping this button.

**Comment by nodiscc** [comment]
This button sits near the search bar "clear" button and i think it is a sensible placement (want to get rid of filtering, move mouse in that area). Above the panels would reduce vertical space there (already quite small with 3 panels), but

> Currently, it doesn't seem intuitive that they change all the panels to "All",

I agree the distance between panels and button is confusing.

> Currently, it doesn't seem intuitive that they change all the panels to "All",

Why not change the wording? `Reset filters`?

**Comment by timpulver** [comment]
>I use this button quite often - when I want to clear all current filters from the left browser panes.

I didn’t know that it was used like that.

>Why not change the wording? `Reset filters?``

Better. But might be confusing because the search bar is also a filter.

**Comment by urielz** [comment]
I agree that "Select All" is not intuitive. Perhaps "Reset Browser"? Although I'm not very fond of large text..

**Comment by nodiscc** [comment]
Or simply move the Select All and gear buttons to the left of the search bar?
